File: test-suite/src/test/java/io/micronaut/guice/doc/examples/bindings/imported/Application.java
package io.micronaut.guice.doc.examples.bindings.imported;

import io.micronaut.guice.annotation.Guice;
import io.micronaut.runtime.Micronaut;

// tag::guice[]
@Guice(
    modules = CreditCardProcessorModule.class,
    classes = PayPalCreditCardProcessor.class
)
// end::guice[]
public class Application {

    public static void main(String[] args) {
        Micronaut.run(Application.class, args);
    }
}

File: test-suite-kotlin/src/test/kotlin/io/micronaut/guice/doc/examples/bindings/imported/Application.kt
package io.micronaut.guice.doc.examples.bindings.imported

import io.micronaut.guice.annotation.Guice
import io.micronaut.runtime.Micronaut

// tag::guice[]
@Guice(
    modules = [CreditCardProcessorModule::class],
    classes = [PayPalCreditCardProcessor::class]
)
// end::guice[]
class Application {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            Micronaut.run(Application::class.java, *args)
        }
    }
}
